MIGRATING.md

```diff
@@ -933,7 +933,7 @@ StripeIntent.Status.Succeeded
    CustomerSession.getInstance().setCustomerShippingInformation(listener);
    ```
 
-- [`PaymentIntent`](https://github.com/stripe/stripe-android/blob/master/stripe/src/main/java/com/stripe/android/model/PaymentIntent.java) has been updated to reflect [API version 2019-02-11](https://stripe.com/docs/upgrades#2019-02-11)
+- [`PaymentIntent`](https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/model/PaymentIntent.java) has been updated to reflect [API version 2019-02-11](https://stripe.com/docs/upgrades#2019-02-11)
```

Comment:
Did you intend to change the migrating instructions for previous releases?

Reply:
I think so, this would represent the most recent change, otherwise these links won't be avaiable